src/audit-whitelist.json

```diff
@@ -0,0 +1,11 @@
+{
+    "ignore": [
+        { "id": "e7bea77b-f37f-4290-a069-08fc9c4b6ac0", "package": "acorn" },
```

Comment:
Have security vulnerability tickets been raised in the triage column on the backlog for this? If not, could we please raise those before merging this so we don't forget to come back to these.

Reply:
Either way i have added the missing stories to the backlog to cover all these low risk vulnerabilities